zulipterminal/core.py

```diff
@@ -120,6 +122,65 @@ def show_stream_info(self, color: str, name: str, desc: str) -> None:
         show_stream_view = StreamInfoView(self, color, name, desc)
         self.show_pop_up(show_stream_view, "# {}".format(name))
 
+    def show_user_info(self, user_id:int) -> None:
+        user_id = user_id
+        url = 'users/' + str(user_id)
+        result = self.client.call_endpoint(
```

Comment:
It's always good to check if the result was a success or not here. We do that in many places so you can have a look.

Reply:
I checked the other places. They just check if the response message was 'success' or not, with a return statement at the end of the function (`return response['result'] == success`). Do you want me to just write that only? Sorry, if I am wrong, but can you please elaborate on this? I didn't put a try-except here because I think it isn't required.

Reply:
Yup, that's the correct way to do the validation. No need of a try-catch block.